components/list/demo/list-drag-and-drop-position.js

```diff
@@ -5,7 +5,8 @@ import { html, LitElement } from 'lit-element/lit-element.js';
 import { ifDefined } from 'lit-html/directives/if-defined.js';
 import { repeat } from 'lit-html/directives/repeat.js';
 
-class ListDemoDragAndDropUsage extends LitElement {
```

Comment:
Just renaming for some consistency.  I will be doing another PR shortly to cleanup the demos and the docs.